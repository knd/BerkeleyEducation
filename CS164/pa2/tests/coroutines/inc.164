def f(x) { 
    def y = yield(x) 
    x + y
} 

def co = coroutine(f)
print resume(co, 1) 
print resume(co, 2)
print resume(co, 3) 
