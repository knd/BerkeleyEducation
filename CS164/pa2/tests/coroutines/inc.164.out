1
3
Error
